lib/runner/filematcher.js

```diff
@@ -53,15 +53,20 @@ module.exports = {
       var match = true;
 
       if (!Array.isArray(moduleTags)) {
-        return typeof opts.tag_filter == 'undefined' && typeof opts.skiptags != 'undefined';
+        return typeof opts.tag_filter == 'undefined' && typeof opts.tags_filter == 'undefined' && typeof opts.skiptags != 'undefined';
```

Comment:
I think the property name `tags_filter` is too close to the existing one `tag_filter` and could encourage errors and make the code a bit more difficult to read. Perhaps call it `multiple_tags_filter` or something like that?